src/docfx.website.themes/common/RestApi.common.js

```diff
@@ -22,6 +22,29 @@ exports.transform = function (model) {
             resolveAllOf(child);
             transformReference(child);
         };
+        if (!model.tags || model.tags.length == 0) {
```

Comment:
for array.length, which return value is number, I think use == is ok here.

---
In reply to: [68890026](https://github.com/dotnet/docfx/pull/442#discussion_r68890026) [](ancestors = 68890026)